source/Willow-Bootstrap/BootstrapNavigationBar.class.st

```diff
@@ -27,26 +27,26 @@ BootstrapNavigationBar class >> brandedWith: aBrandWebView containing: aWebView
 BootstrapNavigationBar class >> fluidBrandedWith: aBrandWebView containing: aWebView applying: aCommandToComponent [
 
 	^ self
-		styledAs: BootstrapCssStyles containerFluid
+		styledAs: [ :container | container addClass bootstrap containerFluid ]
 		brandedWith: aBrandWebView
 		containing: aWebView
 		applying: aCommandToComponent
 ]
 
 { #category : #'private-instance creation' }
-BootstrapNavigationBar class >> styledAs: aContainerClass brandedWith: aBrandWebView containing: aWebView applying: aCommandToComponent [
+BootstrapNavigationBar class >> styledAs: aContainerStylingCommand brandedWith: aBrandWebView containing: aWebView applying: aCommandToComponent [
```

Comment:
styledAs: is no longer correct. We need to change it to applying, but in some way to keep the other command that was already in place